pkg/unikontainers/hypervisors/firecracker.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 Nubificus LTD.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hypervisors
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+const (
+	FirecrackerVmm    VmmType = "firecracker"
+	FirecrackerBinary string  = "firecracker-"
```

Comment:
We shoud change FirecrackerBinary to `firecracker`.

Reply:
Yes, that is a good point. I have addressed this comment and pushed the changes.